### ISSUE TITLE
Running grunt flow throws Warning: Object #<Object> has no method 'error' Use --force to continue.

### ISSUE THREAD

**Issue by akinnee** [larsonjj/grunt-flow#2]
Running grunt flow throws
`Warning: Object #<Object> has no method 'error' Use --force to continue.`

Config:
```
	{
		psApiClient: {
			src: ['src/*.jsx']
		}
	}
```

**Comment by larsonjj** [comment]
Hey @akinnee, It looks like your configuration is incorrect (not sure why you are using the `src` property). Please use the format communicated in the README:

```js
grunt.initConfig({
  flow: {
    options: {
        style: 'color'
    },
    files: {}
  }
})
```

**Comment by akinnee** [comment]
If you read the grunt docs you'll find that `src` and `files` do the same thing with a different syntax. My config should actually work just fine.

To clarify, I did not specify my entire config. But this is closer:

```
grunt.initConfig({
  flow: {
    psApiClient: { // this is the name of the sub-task and is perfectly valid
        options: {},
        src: ['src/*.jsx']
    }
  }
})
```

FWIW, I switched to https://github.com/isuttell/grunt-flow-type-check and it is working just fine. Thanks for the response though. If it's working for you, then maybe the docs aren't clear enough or I just didn't understand something.
